frontend/src/common/colours.ts

```diff
@@ -2,3 +2,4 @@ export const V1_RED = '#DE5F4C'
 export const V1_BLUE = '#002642'
 export const V1_BACKGROUND = '#FAFAF4'
 export const V1_GREY_BACKGROUND = '#EEEEEE'
+export const REVAMP_GREEN = '#468751'
```

Comment:
Can we rename this to something that might make sense in the long run? maybe just `PRIMARY_GREEN`